packages/sdk-js/src/auth.ts

```diff
@@ -1,11 +1,11 @@
-export const getKeyRequestMessage = (chainId: number, address: string, issuedAt: string, expiredAt: string): string => {
+export const getKeyRequestMessage = (chainId: number, address: string, issuedAt: Date, expiredAt: Date): string => {
```

Comment:
I found a new way in Next.js that don’t require send these parameters through API. They have a Server Action function that could pass variables from client to server with one regular typescript function.

so we could keep the intended data types in parameters, which is very nice.